src/Elastic.CommonSchema.Serilog/EcsTextFormatterConfiguration.cs

```diff
@@ -27,6 +29,7 @@ public class EcsTextFormatterConfiguration : IEcsTextFormatterConfiguration
 		bool IEcsTextFormatterConfiguration.MapCurrentThread { get; set; } = true;
 
 		IHttpAdapter IEcsTextFormatterConfiguration.MapHttpAdapter { get; set; }
+		public ISet<string> LogEventPropertiesToFilter { get; set; }
```

Comment:
nitpick: I think this should be explicitly implemented in line with the other properties, with a fluent method to set, to keep the API consistent.

Reply:
Hey, thanks for the review! I don't think I'm following your line of thought exactly, could you clarify?
Do you mean like:
```suggestion
		public ISet<string> LogEventPropertiesToFilter { get; set; } = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
		public EcsTextFormatterConfiguration FilterLogEventProperties(ISet<string> value) => Assign(this, value, (o, v) => o.LogEventPropertiesToFilter = v);
```
Would still want it to be public to allow manual updates (add, remove, ContainsKey, etc), right? Though I guess technically, you could do updates through the FilterLogEventProperties fluent setter (albeit quite inefficiently)? 

Or am I misunderstanding?

Reply:
No worries! I think the interface property should be explicitly implemented with a fluent method, in line with other properties e.g.

```c#
ISet<string> IEcsTextFormatterConfiguration.LogEventPropertiesToFilter { get; set; }

public EcsTextFormatterConfiguration LogEventPropertiesToFilter(ISet<string> value) => Assign(this, value, (o, v) => o.LogEventPropertiesToFilter = v);
```

I don't know the reasoning why these are fluent methods (a style choice?), but I guess typical usage would be to set once, for the lifetime of the running app, since the configuration is used by `EcsTextFormatter`.

>  Though I guess technically, you could do updates through the FilterLogEventProperties fluent setter (albeit quite inefficiently)?

You would also be able to make changes to the `ISet<string>` assigned by casting to the `IEcsTextFormatterConfiguration` interface.

Reply:
@russcam Done, see https://github.com/elastic/ecs-dotnet/pull/90/commits/8eceb1b2e0518557da6509e01baa6634249c8691